server/fleet/app.go

```diff
@@ -125,6 +130,8 @@ type AppConfig struct {
 	WebhookSettings WebhookSettings `json:"webhook_settings"`
 	Integrations    Integrations    `json:"integrations"`
 
+	MDM MDM `json:"mdm"`
```

Comment:
Let's leave a comment in `AppConfig.Clone` to indicate that this settings sub-section does not require any cloning (it helps see that this was not forgotten, but nothing specific needed to be done).